Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-SaaS_File_External_Access.yml

```diff
@@ -0,0 +1,4472 @@
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 676,
+          "y": 1662
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 6387057f-5c95-46d0-8245-c6102fb214a0
+    type: regular
+    task:
+      id: 6387057f-5c95-46d0-8245-c6102fb214a0
+      version: -1
+      name: Get Drive File Activities
+      description: Retrieves past **file** activity in Google Drive.
+      script: '|||google-drive-activity-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "79"
+    scriptarguments:
+      item_name:
+        simple: items/${Core.OriginalAlert.event.referenced_resource}
+      user_id:
+        simple: ${Core.OriginalAlert.event.google_drive_owner}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1089,
+          "y": 771
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: e2f7ccd1-27d3-49ae-88b2-7a48028d4a94
+    type: title
+    task:
+      id: e2f7ccd1-27d3-49ae-88b2-7a48028d4a94
+      version: -1
+      name: Investigation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "12"
+      - "25"
+      - "69"
+      - "88"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1139
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: c216bd59-7b84-4e73-878b-414ce712a433
+    type: title
+    task:
+      id: c216bd59-7b84-4e73-878b-414ce712a433
+      version: -1
+      name: Users Indicators Enrichment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "19"
+      - "13"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -1846,
+          "y": 1267
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: aa508004-a077-4482-a8cd-ef3b5b3353d0
+    type: condition
+    task:
+      id: aa508004-a077-4482-a8cd-ef3b5b3353d0
+      version: -1
+      name: Has the Actor IP returned?
+      description: In *Drive* alerts, actor IP address is available rarely.
```

Comment:
It's not clear what this means. Should it be Whether the Actor IP address is available